plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryAvroConnectorTest.java

```diff
@@ -90,85 +86,9 @@ public void testSelectFailsForColumnName()
     }
 
     @Override
-    @Test
-    public void testProjectionPushdownReadsLessData()
-    {
-        try (TestTable testTable = new TestTable(
-                getQueryRunner()::execute,
-                "test_projection_pushdown_reads_less_data_",
-                "AS SELECT val AS id, CAST(ROW(val + 1, val + 2) AS ROW(leaf1 BIGINT, leaf2 BIGINT)) AS root FROM UNNEST(SEQUENCE(1, 10)) AS t(val)")) {
-            MaterializedResult expectedResult = computeActual("SELECT val + 2 FROM UNNEST(SEQUENCE(1, 10)) AS t(val)");
-            String selectQuery = "SELECT root.leaf2 FROM " + testTable.getName();
-            Session sessionWithoutPushdown = sessionWithProjectionPushdownDisabled(getSession());
-
-            assertQueryStats(
-                    getSession(),
-                    selectQuery,
-                    statsWithPushdown -> {
-                        DataSize physicalInputDataSizeWithPushdown = statsWithPushdown.getPhysicalInputDataSize();
-                        DataSize processedDataSizeWithPushdown = statsWithPushdown.getProcessedInputDataSize();
-                        assertQueryStats(
-                                sessionWithoutPushdown,
-                                selectQuery,
-                                statsWithoutPushdown -> {
-                                    if (supportsPhysicalPushdown()) {
-                                        assertThat(statsWithoutPushdown.getPhysicalInputDataSize()).isGreaterThan(physicalInputDataSizeWithPushdown);
-                                    }
-                                    else {
-                                        // TODO https://github.com/trinodb/trino/issues/17201
-                                        assertThat(statsWithoutPushdown.getPhysicalInputDataSize()).isEqualTo(physicalInputDataSizeWithPushdown);
-                                    }
-                                    assertThat(statsWithoutPushdown.getProcessedInputDataSize()).isGreaterThan(processedDataSizeWithPushdown);
-                                },
-                                results -> assertThat(results.getOnlyColumnAsSet()).isEqualTo(expectedResult.getOnlyColumnAsSet()));
-                    },
-                    results -> assertThat(results.getOnlyColumnAsSet()).isEqualTo(expectedResult.getOnlyColumnAsSet()));
-        }
-    }
-
-    @Override
-    @Test
-    public void testProjectionPushdownPhysicalInputSize()
+    protected Session withoutSmallFileThreshold(Session session)
```

Comment:
is adding declaration of withoutSmallFileThreshold(Session session) is directly connected with removing those two other methods? if no pls split

Reply:
Yes, It is connected. Both the test methods use `withoutSmallFileThreshold`.